style1.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 50px;
+}
+
+.nav-title {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #28352f;
+  margin-left: 12px;
+  margin-top: 10px;
+  padding: 10px 12px;
+}
+
+.hamburger {
+  margin-right: 20px;
+  align-items: center;
+}
+
+.hamburger img {
+  margin-top: 10px;
+}
+
+.hero {
+  background-image: url(Images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: auto;
+}
+
+.hero-container {
+  width: 100%;
```

Comment:
- [ ] [OPTIONAL] Currently, there is a small horizontal overflow on the page. Kindly fix it for a better user experience. I think removing `width: 100%;` will help in fixing the issue. 👍